src/providers/twitch/PubsubClient.cpp

```diff
@@ -703,16 +726,47 @@ PubSub::PubSub()
                     << "Error parsing moderation action:" << ex.what();
             }
         };
-
-    this->moderationActionHandlers["modified_automod_properties"] =
+    this->channelTermsActionHandlers["delete_blocked_term"] =
         [this](const auto &data, const auto &roomID) {
-            // The automod settings got modified
+            // This term got deleted
             AutomodUserAction action(data, roomID);
+
             getCreatedByUser(data, action.source);
-            action.type = AutomodUserAction::Properties;
-            this->signals_.moderation.automodUserMessage.invoke(action);
+
+            try
+            {
+                action.type = AutomodUserAction::RemoveBlocked;
+                if (!rj::getSafe(data, "text", action.message))
+                {
+                    return;
+                }
+
+                if (!rj::getSafe(data, "requester_login", action.source.name))
+                {
+                    return;
+                }
+
+                this->signals_.moderation.automodUserMessage.invoke(action);
+            }
+            catch (const std::runtime_error &ex)
+            {
+                qCDebug(chatterinoPubsub)
+                    << "Error parsing channel terms action:" << ex.what();
+            }
         };
 
+    // We don't get this one anymore or anything similiar
+    // We need some new topic so we can listen
```

Comment:
Just tested `automod-levels-modification.<user-id>.<channel-id>` & it doesn't work with third-party oauth, so we're SOL on this for now